aqt/metadata.py

```diff
@@ -589,20 +589,26 @@ def fetch_http(rest_of_url: str, is_check_hash: bool = True) -> str:
                     )
 
     def iterate_folders(self, html_doc: str, filter_category: str = "") -> Generator[str, None, None]:
-        def table_row_to_folder(tr: bs4.element.Tag) -> str:
+        def link_to_folder(link: bs4.element.Tag) -> str:
             try:
-                return tr.find_all("td")[1].a.contents[0].rstrip("/")
+                url: str = link.get("href")
+                url = url.split("?")[0].rstrip("/")
```

Comment:
Have you looked at `urllib.parse`? If you import `urlparse` from `urllib.parse`, you can do this:
```suggestion
                url = urlparse(url).path.rstrip("/")
```
... and get the benefit of a thoughtfully-designed URL parsing library that will take out anything else that doesn't belong in the path (for instance, `#some_id`).

I'm not sure we really need to use `urlparse`, but I think it's worth asking.